pkg/webhook/mutatingwebhook/mutate.go

```diff
@@ -0,0 +1,132 @@
+package mutatingwebhook
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/pkg/errors"
+	"k8s.io/api/admission/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var (
+	runtimeScheme = runtime.NewScheme()
+	codecs        = serializer.NewCodecFactory(runtimeScheme)
+	deserializer  = codecs.UniversalDeserializer()
+
+	log = logf.Log.WithName("users_pods_mutating_webhook")
+
+	patchContent = patchedContent()
+)
+
+const (
+	priority          = int32(-10)
+	priorityClassName = "sandbox-users-pods"
+)
+
+func patchedContent() []byte {
+	patchItems := []map[string]interface{}{
+		{
+			"op":    "replace",
+			"path":  "/spec/priorityClassName",
+			"value": priorityClassName,
+		},
+		{
+			"op":    "replace",
+			"path":  "/spec/priority",
+			"value": priority,
+		},
+	}
+
+	patchContent, err := json.Marshal(patchItems)
+	if err != nil {
+		log.Error(err, "unable marshal patch items")
+		os.Exit(1)
+	}
+	return patchContent
+}
+
+func HandleMutate(w http.ResponseWriter, r *http.Request) {
+	var respBody []byte
+	body, err := ioutil.ReadAll(r.Body)
+	defer func() {
+		if err := r.Body.Close(); err != nil {
+			log.Error(err, "unable to close the body")
+		}
+	}()
+	if err != nil {
+		log.Error(err, "unable to read the body of the request")
+		w.WriteHeader(http.StatusInternalServerError)
+		respBody = []byte(fmt.Sprintf("unable to read the body of the request: %s", err))
+	} else {
+		// mutate the request
+		respBody = mutate(body)
+		w.WriteHeader(http.StatusOK)
+	}
+	if _, err := w.Write(respBody); err != nil {
+		log.Error(err, "unable to write response")
+	}
+}
+
+func mutate(body []byte) []byte {
+	log.Info("received", "body", string(body))
```

Comment:
Wouldn't it be too verbos to log the entire request body?

Reply:
yeah, probably - remved in 164bb62